modules/platforms/dotnet/Apache.Ignite.Core.Tests/Client/Cluster/ClientClusterTests.cs

```diff
@@ -199,7 +199,7 @@ public void TestInvalidCacheNameTriggersIgniteClientException()
             const string invalidCacheName = "invalidCacheName";
             TestDelegate action = () => GetClientCluster().EnableWal(invalidCacheName);
             var ex = Assert.Throws<IgniteClientException>(action);
-            Assert.AreEqual("Cache doesn't exist: " + invalidCacheName, ex.Message);
+            StringAssert.Contains("Cache doesn't exist: " + invalidCacheName, ex.Message);
```

Comment:
We did not enable stack traces for this test (and below), so existing behavior should remain unchanged. Please adjust the logic and revert assertion changes in .NET tests.